pkg/didcomm/packager/api.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package packager
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/envelope"
+)
+
+// Package packager manages the handling of DIDComm raw messages in JWE compliant envelopes.
```

Comment:
I think the top level Packager interface is agnostic of JWE?

Reply:
what do you think I should say instead for "DID Exchange flows"? Since this format is used for all protocol communications.

Reply:
I don't see "DID Exchange" - I saw "DIDComm".

Reply:
A few lines down.